lib/apiservers/service/swagger.json

```diff
@@ -803,6 +803,10 @@
           "type": "string",
           "format": "uri",
           "pattern": "^(tc|ud)p:\/\/.*"
+        },
+        "container_name_convention": {
```

Comment:
Based on the comments in `vic-machine/common/container.go`, it seems like we expect more container-related properties to be added in the future:
```
		// other container flags to to added"
		// default container memory
		// default container cpu
		// default container network
```

Given that, I think this should probably be structured along the lines of the following to give us more flexibility to eventually represent those properties as a part of this JSON structure:
```
"container": {
    "type": "object",
    "properties": {
        "name_convention": {
            "type": "string",
            "pattern": "^.*(\\{id\\}|\\{name\\}).*"
        }
    }
}
```

Reply:
good point, thanks